src/MinimumNodeService.cpp

```diff
@@ -147,13 +153,6 @@ void MinimumNodeService::process(UserInterface::RequestedAction requestedAction)
      }
    }
 
-// Renegotiating timed out.  Revert to previous NN   
-   if (renegotiating && !bModeSetup)
-   {
-     controller->sendMessageWithNN(OPC_NNACK);
-     setNormal();
-   }
-     
```

Comment:
I want to move this code into checkModeChangeTimeout() as the comment suggests this is to be run at timeout.

As I understand the condition and comment, if bModeSetup has been cleared, we are no longer in setup. This flag is cleared in checkModeChangeTimeout(). So I think we can move this check for renegotiating into checkModeChangeTimeout().

Reply:
Yes, I think that you're correct but we also need 
renegotiating = false; before
controller->sendMessageWithNN(OPC_NNACK); because you're no longer calling setNormal();

Reply:
Thanks. Missed that. Added now.